### PR TITLE
One-shot tasks

### PR DESCRIPTION
**Pull request by furquan-rivos**

### CODE REVIEW

**Comment by furquan-rivos** [comment]
[This hubris change](https://github.com/rivosinc/hubris/pull/54) goes along with this humility change.

**Review by timw-rivos — DISMISSED** [review]

File: cmd/tasks/src/lib.rs [hunk 1]
```diff
@@ -433,7 +433,7 @@ fn explain_sched_state(
     use doppel::SchedState;
 
     match e {
-        SchedState::Stopped => print!("not started"),
+        SchedState::Stopped => print!("Stopped"),
```
Comment (beezow) [inline cmd/tasks/src/lib.rs]: nit: should this be either all lowercase or all caps to match the style for the other states?

Comment (furquan-rivos) [reply cmd/tasks/src/lib.rs]: Done. Pushed a follow-up change.

**Review by beezow — DISMISSED** [review]

**Comment by furquan-rivos** [comment]
Removed the earlier changes since we no longer reuse the stopped state for exit.

**Review by beezow — APPROVED** [review]

**Pull request merged**